relay-profiling/src/client_sdk.rs

```diff
@@ -0,0 +1,7 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct ClientSdk {
```

Comment:
Short doc comments on the fields and struct would be nice (unfortunately missing on the entire module, but usually we have short doc comments in Relay, feel free to ignore).